packages/components/atoms/f-form-field/src/components/FormSelectionControl.vue

```diff
@@ -0,0 +1,187 @@
+<template>
+    <div
+        :data-test-id="testId.container">
+        <input
+            :id="$attrs.id"
+            :aria-labelledby="`label-${$attrs.id}`"
+            v-bind="attributes"
+            :type="inputType"
+            :value="value"
+            :data-test-id="testId.input"
+            :class="[
+                $style['c-formField-field'],
+                $style['c-formField-field--noFocus'],
+                {
+                    [$style['c-formField-field--checkbox']]: isCheckbox,
+                    [$style['c-formField--invalid']]: hasError
+                }]"
+            @change="updateSelectionControl"
+        >
+
+        <form-label
+            :id="`label-${$attrs.id}`"
+            :label-for="$attrs.id"
+        >
+            {{ labelText }}
+        </form-label>
+    </div>
+</template>
+
+<script>
+import FormLabel from './FormLabel.vue';
+import {
+    VALID_SELECTION_CONTROL_TYPES,
+    DEFAULT_SELECTION_CONTROL_TYPE
+} from '../constants';
+
+export default {
+    components: {
+        FormLabel
+    },
+
+    inheritAttrs: false,
+
+    props: {
+        labelText: {
+            type: String,
+            default: ''
+        },
+        attributes: {
+            type: Object,
+            default: () => {}
```

Comment:
Should return an object rather than void
```suggestion
            default: () => ({})
```

Reply:
I was following `FormDropdown.vue`, but yeah, good suggestion.